codegen/lib/objc-codegen.js

```diff
@@ -0,0 +1,280 @@
+var fs = require('fs');
+var ejs = require('ejs');
+
+/**
+ * Generate iOS Client-side Objective-C representation of the models.
+ *
+ * @param {Object} app The loopback application created via `app = loopback()`.
+ * @returns {string} An array of the generated source file contents.
```

Comment:
AFAICT, the method returns an object (hash map) indexed by file names with file contents as the value. Could you please fix the jsdoc annotation?